mw-webapp/src/service/PathToCollection.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Path to collections on firestore
```

Comment:
does path should be enum? what if some of them become dynamical?

Reply:
How path to collections can be dynamical? 
Should I again create const in each service file ?

Reply:
Done